### PR TITLE
Fix some crappy code

### PR DESCRIPTION
**Pull request by Kennene**
Fix some stuff with some phyton magic

### CODE REVIEW

File: phyton.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import time
+
+print("Program is calculating, please wait...")
+
+while True:
+       time.sleep(1)
```
Comment (Jnkess) [inline phyton.py:6]: Solid changes, thats a great use of sleep

**Review by Jnkess — APPROVED** [review]

File: phyton.py (hunk 1 above)
Comment (Kennene) [reply phyton.py:6]: nah, it's bloated, had to import whole library for that

**Comment by Jnkess** [comment]
@dlisiewicz Can I please get a write permission to merge this PR? I think we need to go live with this change immediately!!

File: phyton.py (hunk 1 above)
Comment (Jnkess) [reply phyton.py:6]: Yes, but I jest looked into logs, and we got 3 times faster response time, and saved up some space in cloud. Dont be so strict on yourself Krzysztof. Kudos to you, great work!!

**Pull request closed without merging**